fiori: add warning to protect partial updates (#2641)
cap-js/docs: guides/uis/fiori.md

> Directly updating an active entity does **not** bypass [draft locks](#draft-locks).
> If an existing draft locks the entity, direct updates are blocked to prevent lost update situations.

> > [!warning] Ensure additional entry points are imlemented

Reviewer: Should we mention validation in the title?

```suggestion
> [!warning] Ensure validation for all entry points
```

> > Through this feature also partial CREATE/UPDATE requests to the root entity and its composition children have to be handled. Ensure that the validations and determinations are run in all situations, not only on the root.

Reviewer: ```suggestion
> Requests to active data also features partial CREATE/UPDATE requests to the root entity and its composition children. Ensure that the validations and determinations are run in all situations, not only on the root.
```
